transforms/button-destructive/__testfixtures__/SpreadOverride.output.js

```diff
@@ -1,6 +1,6 @@
 import Button from "carbon-react/lib/components/button";
 const props = {
-  type: "primary",
+  buttonType: "primary",
```

Comment:
I would prefer single quotes in JS code, maybe its worth updating?

Reply:
All the formatting in this project is handled by `prettier`.

As for the transform itself `esprima` will work out what the file is currently using and will keep the existing format I believe.